### DISCUSSION TITLE
Requirements

### DISCUSSION THREAD

**Discussion by aali0906** [petrsvihlik/WopiHost#120] [Q&A]
Hi Everybody,

does this also work with Microsoft 365 plans or do I  necessarily need OOS?

Regards

**Comment by petrsvihlik** [comment]
Hey @aali0906,
I test it using OOS2016. I actually don't have access to the [Office Cloud Storage Partner Program](https://developer.microsoft.com/en-us/office/cloud-storage-partner-program) yet. I'm working on that and hope it will change soon.
One thing I recommend doing is running WopiHost against the [WOPI Validator](https://wopi.readthedocs.io/en/latest/build_test_ship/validator.html#validator) which should give you a reasonably accurate answer. It is something I'm [planning on doing](https://github.com/petrsvihlik/WopiHost/issues/74) myself by the end of April.
